examples/exececution-data/execution-record.json

```diff
@@ -0,0 +1,43 @@
+{
+  "summary": {
+    "name": "Example Boutiques Tool",
+    "descriptor-doi": "10.5281/zenodo.2634310",
+    "date-time": "2019-03-18T07:26:12.459313"
+  },
+  "public-invocation": {
+    "str_input_list": [
+      "fo '; echo FAIL",
+      "bar"
+    ],
+    "str_input": "coin;plop",
+    "file_input": {
+      "file-name": "setup.py",
+      "hash": "cfca29dcf1578b0d656e8a4e4b8964ca"
+    },
+    "list_int_input": [
+      1,
+      2,
+      3
```

Comment:
Would the code hash each element in a list of files?

Reply:
This isn't a list of files, its a list of int value inputs.

Reply:
Fixed